docs/source/tutorials/using-your-kit.rst

```diff
@@ -0,0 +1,281 @@
+   do stuff, so make sure to have a look at it first!
+
+Setup
+--------
+Before you start this tutorial, make sure you have `connected your 
+kit together`_.
+
+.. _connected your kit together: connecting-your-kit
+
+There are two lines of code you must have at the very top of your code whenever 
+you want to do something with the robot, those lines are:
+.. code:: python
+    from sbot import *
+
+    r = Robot()
+
+These two lines simply copy in all of our helper functions, and sets up your 
+robot.
+
+Any code below the line with ``r = Robot()`` won't be run until you hit the
+black 'Start' button on the power board.
+
+Forwards and Backwards
+----------------
+
+Start by checking that you can drive your motors forwards and backwards.
+Doing this is actually very easy; the only thing you need to realise is that a
+positive number is forwards and a negative number is backwards.
+
+.. Warning:: Make sure your robots can turn without danger.
+   If your motors aren't attached to a chassis, make sure they don't have wheels
+   and are sitting in a position which makes it safe for them to turn.
+
+Please remember that the actual movement of the robot depends which way around
+the motor is mounted on the robot! It's very likely you might need to set one
+motor to go in reverse in order for your robot to go forwards.
+
+Here's the code:
+
+.. code:: python
+
+    from sbot import *
+    from time import sleep
+    while True:
+        r = Robot()
+        # Set motor 0 to 20% power.
+        r.motor_board.motors[0].power = 0.2 
+        # Set motor 1 to 20% power
+        r.motor_board.motors[1].power = 0.2 
+
+        sleep(1)
+
+        r.motor_board.motors[0].power = 0
+        r.motor_board.motors[1].power = 0
+
+        sleep(1)
+
+You’re hopefully familiar with the first few lines; in fact, the only lines you 
+may not be familiar with are the r.motors_board.motors[0]... lines. For a
+comprehensive reference to the 'motor' object, see `the motor page 
+</en/latest/api/motor-board.html>`_. 
+
+But, to summarise:
+
+.. code:: python
+
+    r.motor_board.motors[0].power = x
+
+Will set the power of the motor connected to output will set the power of the
+motor connected to output 0 (the ``motors[0]`` part) on the motor board to 
+``x``, where ``x`` is a value between ``-1`` and ``1``, inclusive.
+
+Now see if you can turn on the spot, then try driving in various shapes.
+
+Write some code that would make your robot drive in:
+- a square
+- a wavy line
+
+Changing the speed
+------------------
+
+When you move your robot, it's likely you'll want your robot to go from stand-
+still to moving at a high speed. The dumb way of doing this is to just immediately
+set the power of the motors from ``0`` to ``1``.
+
+There are 2 problems with doing this:
+
+1. Setting the power from ``0`` to ``1`` very quickly draws a very large current
+   from the motor, So much so that it might trip the safety on our power board.
```

Comment:
Fix typo and use clearer wording

```suggestion
   from the motor, so much so that it might trip the over-current protection on our power board.
```